package.json

```diff
@@ -219,6 +223,7 @@
   "dependencies": {
     "@popperjs/core": "2.11.0",
     "@reach/menu-button": "0.16.2",
+    "@storybook/client-api": "6.4.10",
```

Comment:
* Is this needed?
* If so, is it needed as a dependency?

Reply:
not needed